### DISCUSSION TITLE
LineageOS needs your help :)

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#23312] [Q&A]
LineageOS just rolled out new update for external chromium webview:  https://github.com/LineageOS/android_external_chromium-webview_prebuilt_arm64/commits/main

Now while building LineageOS-20.0 its not properly syncing with --git-lfs resulting in this error everytime: https://cirrus-ci.com/task/6024317801070592?logs=build#L6141

Sir, you have already fixed this issue last year now same thing happening once again, please do check it out & BlissRoms-Typhoon also needs to be synced with --git-lfs as they track chromium webview from LineageOS, if not then it throws the same error: https://cirrus-ci.com/task/5887977537142784?logs=build#L5543

**Comment by Apon77** [comment]
what is the sync or init command they need?? please put exactly

**Reply by SwapnilVicky** [reply]
we need this init command: repo init -u https://github.com/LineageOS/android.git -b lineage-20.0 --git-lfs   

with --git-lfs at the end, this is missing

**Reply by Apon77** [reply]
but you already can use --git-lfs, script don't block that. what is the solution then?

**Reply by ghost** [reply]
I think the issue is with Lineage's latest  WebView "main" branch because when I pulled "backup/main_20230314" instead of main the build worked fine.

**Reply by SwapnilVicky** [reply]
i have --git-lfs already in my init script but it doesnt work, dont know if we have any other options to fix

**Reply by wHo-EM-i** [reply]
git-lfs already installed?
> sudo apt install -y git-lfs

**Reply by Apon77** [reply]
git-lfs is installed

**Reply by SwapnilVicky** [reply]
git-lfs is not working now, its working earlier without the --git-lfs tag in init script
